src/site/stages/build/options.js

```diff
@@ -51,7 +51,6 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
     defaultValue: process.env.DRUPAL_PASSWORD,
   },
   { name: 'no-drupal-proxy', type: Boolean, defaultValue: false },
-  { name: 'local-proxy-rewrite', type: Boolean, defaultValue: false },
```

Comment:
The TeamSite proxy needs to be started separately now, so that we can start the proxy during e2e tests.

Reply:
Ah, yes 🧙